src/com/theironyard/novauc/Person.java

```diff
@@ -0,0 +1,31 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by Eric on 2/16/17.
+ */
+public class Person {
+
```

Comment:
Eric good job man.  One thing, as your code is right now, nothing is being written to the json file.  This is because the json serializer uses getters and setters to access the information from the person class.  Without those it can't do it's job.  I'm sure that you can handle generating getters and setters so there is no need to resubmit.  Again good job.  You should be proud of the progress you are making.